src/components/Footer.tsx

```diff
@@ -1,5 +1,92 @@
-const Footer = () => {
-  return <div className="debug h-[var(--footer-height)]">我是 Footer</div>;
+import { Button } from '@components/ui/button';
+import cn from '@lib/utils';
+import { ArrowUpRight } from 'lucide-react';
+import Image from 'next/image';
+import Link from 'next/link';
+import logo from '../../public/Logo.svg';
+
+type FooterProps = {
+  className: string;
+};
+
+const SITEMAP = [
+  {
+    name: '探索活動',
+    url: '/explore',
+  },
+  {
+    name: '推薦活動',
+    url: '/recommended',
+  },
+  {
+    name: '常見問題',
+    url: '/faq',
+  },
+  {
+    name: '關於我們',
+    url: '/about',
+  },
+];
+
+const Footer = ({ className }: FooterProps) => {
+  return (
+    <footer
+      className={cn(
+        'space-y-12 px-3 py-24 text-primary xl:space-y-24 xl:pb-36 xl:pl-[312px]',
```

Comment:
```suggestion
        'space-y-12 px-3 py-24 text-primary xl:space-y-24 xl:pb-36 xl:max-w-[81rem]',
```

然後要到layout.tsx，設定成  
```js
    <body
      className={cn(
        'container min-h-screen bg-background font-sans antialiased ',
        fontSans.variable
      )}
    >
      <AuthProvider>
          /* something else */
        <Footer className="mx-auto" />
      </AuthProvider>
    </body>
 ```
 
 只是說要動到layout.tsx，我想說晚點等大家同意再修改也行

Reply:
好的，沒問題！非常感謝～
另外想請問 `max-w-[81rem]` 的部分，81rem 為 1296px，如果設置成 80rem = 1280px 的話，會不會比較符合其他的斷點 xl 寬度？

Reply:
嗯，其實不是斷點。

你看figma設計稿，上面的寬度全部都是1296px

再來就是斷點1280是指以上，所以也沒問題的。

還有其實不設定`xl:max-w-[81rem]`也是可以正常運作，只是擔心以後footer多加東西，先預留而已。

Reply:
了解，謝謝🙌🙌

Reply:
再等mia幫你看囉，我這邊okay了

Reply:
剛剛沒注意到，再補充一個`xl:px-0`

Reply:
好的好的！感謝～～

Reply:
沒問題～不過如果最後決定統一斷點的話，記得去tailwind.config改

Reply:
看起來是少了 vercel-token 的樣子，這可能要再討論當初的設定確認一下～

另外，因為剛好還沒有 merge 進去，還有這部分的更動我還沒 commit 上去，想說等這邊也確定好是否更新 layout.tsx 和斷點後再修改（目前畫面在 1280px-1296px 沒有 padding）到時再請 @miamai review，看這樣可以嗎？

Reply:
缺少vercel token應該是因為現在的github flow用fork專案產生的，星期一會再問問大家，不要用fork專案的github flow跑法

Reply:
另外layou.tsx也可以晚點再改，想像成你這次製作的區域，和layout.tsx修正的區域並不同就好了

Reply:
好的，那我 Footer 的部分，應該可以先把 `xl:max-w-[81rem] xl:space-y-24 xl:px-0` 加上去